docs/docs/installation.md

```diff
@@ -47,6 +47,10 @@ While React [can be used](/react/docs/react-without-es6.html) without a build pi
 
 ### Installing React
 
+>**Note:**
+>
+>Once installed, we strongly recommend setting the [production mode flags](/react/docs/optimizing-performance.html) to ensure you're using the fast version of React in production.
```

Comment:
Maybe let's say "set up a separate production build process" or something like this? To indicate you don't need to do it for development, but that you need to separate how you develop from how you deploy.